docker-compose.yml

```diff
@@ -224,6 +224,22 @@ services:
     container_name: traefik
     restart: always
     command:
+      - '--log.format=json'
```

Comment:
It doesn't seem to work if use with `docker-compose.https.yml`.

Reply:
Good catch. Fixed.